cmd/jaeger/internal/extension/jaegerquery/server.go

```diff
@@ -52,7 +55,13 @@ func (*server) Dependencies() []component.ID {
 }
 
 func (s *server) Start(ctx context.Context, host component.Host) error {
-	mf := otelmetrics.NewFactory(s.telset.MeterProvider)
+	var meterProvider metric.MeterProvider
+	if s.telset.LeveledMeterProvider != nil {
+		meterProvider = s.telset.LeveledMeterProvider(configtelemetry.LevelNormal)
+	} else {
+		meterProvider = noop.NewMeterProvider()
+	}
```

Comment:
i don't believe you need to do this - you should just be able to pass down the meter provider that you get from OTEL's  telemetry settings

Reply:
I dont think i need to do that here , All the test are working fine with just passing it down .